src/notifications.ts

```diff
@@ -0,0 +1,67 @@
+import { ChannelType, ThreadAutoArchiveDuration } from 'discord.js';
+import { getChannelFromSettings } from '@/util';
+import { NotificationThread } from '@/models/notificationThreads';
+import type { AllowedThreadTypeForTextChannel, BaseGuildTextChannel, Guild, GuildTextThreadCreateOptions, MessageCreateOptions, MessagePayload, ThreadChannel, User } from 'discord.js';
```

Comment:
Very long imports like this should be broken up into multiple lines
```suggestion
import type {
	AllowedThreadTypeForTextChannel,
	BaseGuildTextChannel,
	Guild,
	GuildTextThreadCreateOptions,
	MessageCreateOptions,
	MessagePayload,
	ThreadChannel,
	User
} from 'discord.js';
```